config/examples/Two Trees/Sapphire Pro/Configuration.h

```diff
@@ -1082,15 +1098,23 @@
 // Invert the stepper direction. Change (or reverse the motor connector) if an axis goes the wrong way.
 #define INVERT_X_DIR true
 #define INVERT_Y_DIR true
-#define INVERT_Z_DIR false
+#if ENABLED(SPRO_TMC2209)
+  #define INVERT_Z_DIR true
+#else
+  #define INVERT_Z_DIR false
+#endif
 
 // @section extruder
 
 // For direct drive extruder v9 set to true, for geared extruder set to false.
 #ifdef SPRO_INVERTED_E
-  #define INVERT_E0_DIR true
+  #if ENABLED(SPRO_TMC2209)
+    #define INVERT_E0_DIR false
+  #else
+    #define INVERT_E0_DIR true
+  #endif
 #else
-  #define INVERT_E0_DIR false
+  #define INVERT_E0_DIR true
```

Comment:
Are you sure that you want to flip this value? Seems like this should remain false.

Reply:
you are right